tests/page_test.go

```diff
@@ -440,12 +440,13 @@ func TestPageSetExtraHTTPHeaders(t *testing.T) {
 func TestPageWaitForFunction(t *testing.T) {
 	t.Parallel()
 
+	// script is here to test we're not getting an error from the
+	// waitForFunction call itself and the tests that use it are
+	// testing the polling functionality—not the response from
+	// waitForFunction.
 	script := `
-        page.waitForFunction(%s, %s, %s).then(ok => {
-            log('ok: '+ok);
-        }, err => {
-            log('err: '+err);
-        });`
+		let resp = page.waitForFunction(%s, %s, %s)
```

Comment:
Do we not need `await` here? Otherwise it could print `ok: `, no?

Reply:
❤️